packages/contracts/CONTRIBUTING.md

```diff
@@ -80,6 +83,10 @@ brief explainer of the env variables:
   that is attached to a KPI token that has its expiration at t40 must have a
   measurement timestamp of at maximum t10. Set this to a sufficiently large
   value for production deployments.
+- `ETHERSCAN_API_KEY`: the Etherscan (or Blockscout) API key used to verify
+  contracts.
+- `VERIFIER_URL`: the Etherscan pr Blockscout API URL that will be used to
```

Comment:
Typo here? `pr`?

Reply:
Addressed in 668618de24d6ea47665c8e9bb87b264b189504f4